### PR TITLE
Make G29's Interactive Mesh Editor work.

### PR DESCRIPTION
**Pull request by Roxy-3D**
This Pull Request will make G29's Interactive Mesh Editor work.

### CODE REVIEW

**Comment by 3dGriff** [comment]
Thanks @Roxy-3D I'll merge and deploy on the test machine, be back at the shop this evening, just cracking open the UBL commit. Strong work BTW

**Comment by thinkyhead** [comment]
~~A variable `G29_encoderDiff` is used but not defined.~~ nevermind, found it…

You'll want to rebase your work-branch to `RCBugFix` after my next patch (#6075), or just sync your `RCBugFix` (assuming you're keeping it clean) and make a copy of the updated `RCBugFix` branch in your working copy.

**Comment by thinkyhead** [comment]
> Roxy-3D:RCBugFix 

Ah, I see you're modifying your copy of `RCBugFix`. It's better to never modify it, so you can keep it in sync with the upstream, and always do work in a copy.

**Comment by Roxy-3D** [comment]
Well...   Unfortunately...   I'm going to have to delete my repository and re-create it.   I've totally mangled it.   Git is anything but intuitive.   

Right now...  I'm just keeping lots of backups on my local machine.  And the fact it is merged into RC-8's RCBugFix gives me a pretty current fall back scenario if something really bad happens.

I'm trying to use Git to get stuff done...   But it is hard!

**Comment by thinkyhead** [comment]
Git is quite nice once you've mastered it. I will make a video demonstrating my workflow.

**Comment by 3dGriff** [comment]
@thinkyhead I would be interested in seeing this as well.  I'm trying to correctly spin git into my workflow and struggling a bit.

File: Marlin/ultralcd.cpp [hunk 1]
```diff
@@ -877,13 +879,15 @@ void kill_screen(const char* lcd_msg) {
 
     float lcd_z_offset_edit() {
       lcd_goto_screen(_lcd_z_offset_edit);
+      defer_return_to_status = true;
```
Comment (thinkyhead) [inline Marlin/ultralcd.cpp:882]: I removed these `defer_return_to_status = true` lines from cases like this one, where it's already jumping straight to a screen handler that already sets `defer_return_to_status = true`.

**Pull request merged**